pallets/msa/src/replay_tests_2.rs

```diff
@@ -0,0 +1,171 @@
+use crate::{self as pallet_msa, types::EMPTY_FUNCTION, AddKeyData, AddProvider, Config, Error};
```

Comment:
Not sure how else to do proposed boundary checks which require different configs.  Maybe the filename could be renamed though.